java-file-formats/word-library/convert-markdown-to-word-document-in-java.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Convert Markdown to Word document in Java | DocIO | Syncfusion
+description: Convert Markdown to Word document in java using Syncfusion Java Word library without Microsoft Word or interop dependencies
+platform: java-file-formats
+control: Word library
+documentation: UG
+---
+# Markdown to Word Conversion
+
+Markdown is a lightweight markup language that adds formatting elements to plain text documents. The Java Word library supports the conversion of Markdown to Word document and vice versa, which mostly follows the CommonMark specification and GitHub-flavored syntax.
+
+## Convert Markdown to Word document
+
+Convert an existing markdown file to a Word document (DOCX and RTF) using the Java Word library.
+
+The following code example shows how to convert Markdown to Word document.
+
+{% tabs %}
+{% highlight JAVA %}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+T> You can also save the markdown file as [HTML](https://help.syncfusion.com/file-formats/docio/html).
+
+## Customize image data
+
+The Java Word library provides a MdImageNodeVisited event, which customizes image data while importing a Markdown file. Implement the logic to customize the image data by using this MdImageNodeVisited event.
```

Comment:
The Java Word library provides a ImageNodeVisited event, which customizes image data while importing a Markdown file. Implement the logic to customize the image data by using this ImageNodeVisited event.